### DISCUSSION TITLE
Stumpwm Video Showcase/Demo

### DISCUSSION THREAD

**Discussion by azzamsa** [stumpwm/stumpwm#1107] [Ideas]
It would be great if Stumpwm has video showcase. So the new user can see what features they get and how stumpwm looks like. 

I know this work often done  'unofficialy'. But as today, I don't find any new stumpwm demo.
Two of demos in youtube are 9 and 6 years ago.

Thanks for this great wm.

**Comment by dmb2** [comment]
Hi @azzamsa,

I love this idea! I really want to do something like this, but I don't have the skills.  I will try to look into it, but if you have any inkling please do.  We can set up a official youtube channel and post it there.

**Comment by VanLaser** [comment]
Maybe inspiring a little from this: https://www.youtube.com/watch?v=do0DVxy4HBc

**Comment by equwal** [comment]
Here is a high quality video: https://www.youtube.com/watch?v=4pTSgMlpyzc

Although it is not a positive review. Here is his first sentence:
"Possibly the most frustrating window manager I've ever used; StumpWM."
